cms/static/js/views/partition_group_details.js

```diff
@@ -21,29 +21,39 @@ define([
 
             return [
                 'collection',
-                'content-group-details',
-                'content-group-details-' + index
+                'configuration-group-details',
+                'configuration-group-details-' + index
             ].join(' ');
         },
 
         editGroup: function() {
-            this.model.set({'editing': true});
+            this.model.set({editing: true});
         },
 
         initialize: function() {
-            this.template = this.loadTemplate('content-group-details');
+            this.template = this.loadTemplate('configuration-group-details');
+            this.restrictEditing = this.options.restrictEditing || false;
+            this.scheme = this.options.scheme || 'cohort';
             this.listenTo(this.model, 'change', this.render);
         },
 
         render: function(showContentGroupUsages) {
-            var attrs = $.extend({}, this.model.attributes, {
+            var groupTypeName = 'content group',
```

Comment:
I just looked at how this is used, and I'd like to avoid introducing this complexity (also note groupTypeName is not translated).